src/util/address.rs

```diff
@@ -571,15 +571,15 @@ impl fmt::Display for Address {
                     Network::Testnet | Network::Signet => "tb",
                     Network::Regtest => "bcrt",
                 };
-                let bech_ver = if ver.to_u8() > 0 {  bech32::Variant::Bech32m } else { bech32::Variant::Bech32 };
+                let bech_ver = if version.into_num() > 0 {  bech32::Variant::Bech32m } else { bech32::Variant::Bech32 };
```

Comment:
Just realized, I believe `Copy` types tend to use `to_` instead of `into_`. Not a big deal IMO.

Reply:
I'm a bit confused over this, since https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv explicitly specifies that `to_*` operation is _expensive_ and takes _borrowed_ value, while `into_*` us _unexpensive_ and consumes the value, which is perfect for `Copy` types

Reply:
Looks like that docs is confusing as copying `u8` is considered inexpensive. :)

`into_` says it's for non-Copy types. :man_shrugging: